crates/aptos-crypto/src/bls12381/bls12381_sigs.rs

```diff
@@ -177,6 +178,12 @@ impl Length for Signature {
     }
 }
 
+impl fmt::Display for Signature {
```

Comment:
probably Debug should have the same format

Reply:
Why?

I'm just going by the same `Display` implementation for `bls12381::PublicKey` from `bls12381_keys.rs`:
```
impl fmt::Display for PublicKey {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(f, "{}", hex::encode(&self.to_bytes()))
    }
}
```

There, `bls12381::PublicKey` has a `#[derive(Debug)]`

Reply:
no one can read the derived Debug format, the hex encoded format doesn't lose any information too?

Reply:
Fixed.